### ISSUE TITLE
NYC handlebars dependency warning

### ISSUE THREAD

**Issue by jungleBadger** [istanbuljs/nyc#991]
## Expected Behavior
Don't depend on blacklisted versions of libraries

## Observed Behavior
npm commands will raise a `High` warning through nsp/audit process regarding the Istanbul reports / Handlebars dependency

## Forensic Information
https://www.npmjs.com/advisories/755
https://github.com/istanbuljs/istanbuljs/issues/293

**Operating System:** the operating system you observed the issue on.
Windows 10 / OS X

```
  High            Prototype Pollusion                                           
                                                                                
  Package         handlebars                                                    
                                                                                
  Patched in      >=4.0.13                                                      
                                                                                
  Dependency of   nyc [dev]                                                     
                                                                                
  Path            nyc > istanbul-reports > handlebars                           
                                                                                
  More info       https://npmjs.com/advisories/755               
```

**Comment by Xilis** [comment]
handlebars is a dependency in istanbul-reports

An issue can be found at https://github.com/istanbuljs/istanbuljs/issues/293 ,
and a pullrequest is already made and passed testing at https://github.com/istanbuljs/istanbuljs/pull/294

**Comment by coreyfarrell** [comment]
`nyc@13.3.0` is published with the updated handlebars.  The master branch of nyc already included a feature so for the moment 13.3.0 is not `latest`, it's `next`.  You can install with `npm i -D nyc@next`.  Will be a much shorter testing cycle than normally but I still need to give a bit of time to make sure the new feature doesn't create major issues.

I'm keeping this issue open until nyc is upgraded to `latest` on npm.

**Comment by coreyfarrell** [comment]
nyc@13.3.0 is promoted to `latest` so this issue is now fully resolved.
